pkg/operator/status.go

```diff
@@ -185,6 +185,12 @@ func (optr *Operator) getOrCreateClusterOperator() (*osconfigv1.ClusterOperator,
 			Resource: "namespaces",
 			Name:     optr.namespace,
 		},
+		{
```

Comment:
sweet!
can we ad a PR desc?
can you confirm "" brings all the existing objects?
can we please also add machineSets?

Reply:
@enxebre https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_machine-api-operator/391/pull-ci-openshift-machine-api-operator-master-e2e-aws/1448/artifacts/e2e-aws/must-gather/namespaces/openshift-machine-api/machine.openshift.io/

You can see we're gathering machines and machinesets here.